tool/gravity/cli/install.go

```diff
@@ -683,6 +704,10 @@ func completePlanFromService(
 	operation ops.SiteOperation,
 	connecting, connected string,
 ) error {
+	strategy, err := newResumeStrategy(utils.Exe.WorkingDir)
```

Comment:
Could you please explain a bit what kind of impact this change has on UX? Are all of these commands now depend on the working dir of the binary somehow?

I.e. will users still be able to just execute `gravity join`, `gravity plan`, etc. commands from wherever on cluster nodes?

Reply:
No, I'm simply pulling the previously implicit state configuration to the top so it's easier to change if necessary - there is no change in behavior.